protostar/starknet/cheatable_cached_state.py

```diff
@@ -45,6 +45,9 @@ def __init__(
         self.expected_contract_calls: dict[
             Address, list[tuple[SelectorType, CairoOrPythonData]]
         ] = {}
+        self.expected_contract_calls_cairo: dict[
+            Address, list[tuple[str, CairoOrPythonData]]
+        ] = {}
```

Comment:
Why do we need two maps with the same purpose 🤔

Reply:
Ye, I wrote above that they should be merged into just one